src/app/reducers/environments.ts

```diff
@@ -17,16 +18,27 @@ export interface State {
 }
 
 export const getInitialEnvironmentState = (): EnvironmentState => {
+  const { initialData } = getAltairConfig();
   return {
     title: 'Environment',
-    variablesJson: '{}'
+    variablesJson: '{}',
+    id: uuid(),
+    ...initialData.environments.base
   };
 };
 
+const getInitialSubEnvironmentState = (): EnvironmentState[] => {
+  const { initialData } = getAltairConfig();
+  return (initialData.environments.subEnvironments || []).map(env => {
+    env.id = uuid();
+    return env;
```

Comment:
Same here, I think we should be more explicit as well. Once the data is set, Altair persists it so if we don't sanitize it on entry, it could lead to confusing errors (with confusing issues created on GitHub 😣) in the future.